builder-support/debian/authoritative/debian-buster/control

```diff
@@ -1,40 +1,30 @@
 Source: pdns
 Section: net
-Priority: extra
-Standards-Version: 4.1.1
 Maintainer: PowerDNS Autobuilder <powerdns.support@powerdns.com>
-Origin: PowerDNS
-Build-Depends: autoconf,
-               automake,
-               autotools-dev,
-               bison,
+Priority: optional
+Standards-Version: 4.5.1
+Build-Depends: bison,
                curl,
-               debhelper (>= 10),
+               debhelper-compat (= 10),
                default-libmysqlclient-dev,
-               dh-autoreconf,
-               dh-systemd,
+               dnsutils,
                flex,
-               libboost-dev,
-               libboost-program-options-dev,
-               libboost-serialization-dev,
-               libboost-test-dev,
+               libboost-all-dev,
                libcdb-dev,
                libcurl4-openssl-dev,
                libgeoip-dev,
                libkrb5-dev,
                libldap2-dev,
                liblmdb-dev,
-               libluajit-5.1-dev [!arm64],
-               liblua5.3-dev [arm64],
+               libluajit-5.1-dev [amd64] | liblua5.1-0-dev,
+               liblua5.1-0-dev [!amd64],
```

Comment:
does this move to plain Lua on non-AMD64?

Reply:
yes - kpfleming did that for the ARM64 case

Reply:
oh, but there's a downgrade from 5.3 to 5.1 here for non-amd64!

Reply:
So, this should do:
- on amd64, force luajit
- on other archs, use luajit if its there
- then use lua 5.1. 

I figured its smarter to use 5.1 across all packages and products. Can certainly put 5.3 back in.

Reply:
> I figured its smarter to use 5.1 across all packages and products. Can certainly put 5.3 back in.

@Habbie, your call

Reply:
5.3 please